docs/modules/deployment/pages/upgrade/disaster-recovery/introduction.adoc

```diff
@@ -0,0 +1,36 @@
+[[restore]]
+= Restoring OpenNMS {page-component-title} 
+
+OpenNMS {page-component-title} binaries, logs, RRD archives, configurations and the database will be restored.
+NOTE: In this example the backup storage path is `/backup` for demonstration purposes.
+
+[[restore-objectives]]
+== Objectives
+
+* Restore backup of OpenNMS {page-component-title} components
+* Install PostgreSQL database and restore backup
+* Install OpenJDK 
+* Create systemd OpenNMS service unit
+
+[[restore-requirements]]
+== Requirements
+
+* The backups that were made in [[backup-debian-ubuntu]]
```

Comment:
```suggestion
* The backup that was made in [[backup-debian-ubuntu]]
```
Using "backup" singular, as I see it as a "system" backup, even though there are component backups involved. But open to feedback if you feel plural is better.